src/main/java/uk/gov/hmcts/divorce/divorcecase/tab/CaseTypeTab.java

```diff
@@ -553,6 +555,9 @@ private void buildLetterPackTab(ConfigBuilder<CaseData, State, UserRole> configB
     private void buildChangeOfRepresentativeTab(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
         configBuilder.tab("changeOfRepresentatives", "Change of representatives")
                 .forRoles(CASE_WORKER, SUPER_USER)
+                .field("nocWhichApplicant", NEVER_SHOW)
+                .field("changeOrganisationRequestField", NEVER_SHOW)
+                .showCondition(NOTICE_OF_CHANGE_HAS_BEEN_APPLIED)
```

Comment:
changeOfRepresentatives has a default value (empty list), which means the showCondition we normally use for a missing field does not work. Caseworker NoC sets nocWhichApplicant and common component NoC leaves a blank changeOrganisationRequestField in the case data, so I've checked for the presence of those fields instead to determine whether a NoC has been run and the tab should be visible.